proxysql/metadata.csv

```diff
@@ -0,0 +1,73 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+proxysql.active_transactions,gauge,,transaction,,Count of how many client connection are currently processing a transaction,0,proxysql,Active transactions
+proxysql.query_processor_time_nsec,rate,,nanosecond,,The time spent inside the Query Processor to determine what action needs to be taken with the query (internal module),-1,proxysql,Query processor time
+proxysql.questions,rate,,question,second,Total number of client requests / statements executed,0,proxysql,Questions
```

Comment:
For the descriptions, replace all `Total number of` and `Number of` with `The total number of` and `The number of`.